.github/workflows/main.yml

```diff
@@ -0,0 +1,34 @@
+name: AI Code Reviewer
+
+on: [pull_request]
+
+    # types:
+    #   - opened
+    #   - synchronize
+# permissions: write-all
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      pull-requests: write
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v3
+        
+      - name: Read Review Guidelines
+        id: read-guidelines
+        run: |
+          # Assuming your guidelines are in a file named REVIEW_GUIDELINES.md
+          REVIEW_GUIDELINES=$(cat REVIEW_GUIDELINES.md | base64)
+          echo "::set-output name=review_guidelines::$REVIEW_GUIDELINES"
+
+      - name: AI Code Review Action
+        uses: freeedcom/ai-codereviewer@v2.6.0
+        with:
+          GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
+          OPENAI_API_KEY: ${{ secrets.OPENAI_API_KEY }}
+          OPENAI_API_MODEL: "gpt-4" # Optional: defaults to "gpt-4"
+          exclude: "**/*.json, **/*.md" # Optional: exclude patterns separated by commas
```

Comment:
The `exclude` pattern seems to be incorrect. It should be `**/*.json,**/*.md` without spaces after the comma.